utils/buttons.py

```diff
@@ -466,7 +464,7 @@ async def wait_for_message(self) -> AsyncGenerator[bool, discord.Message]:
                 if value is False:
                     error = self.invalid_response()
                     await message.reply(error, delete_after=60)
-                yield
+                yield None
```

Comment:
y do you want to explicitly yield None tho

Reply:
because mypy wants me to. same for blank returns. if you have optional, you must explicitly use None

Reply:
ya blaming mypy when this repo doesnt even have mypy config

Reply:
you want me to remove it?

Reply:
yes, I dont follow mypy bs, i will accept this if and only if I have a mypy config in the repo for consistency, but I dont.